test/e2e/upgrade.go

```diff
@@ -60,6 +60,18 @@ func runSimpleUpgradeFlowWorkerNodeVersionForBareMetal(test *framework.ClusterE2
 	test.ValidateHardwareDecommissioned()
 }
 
+func runInPlaceMultipleUpgradesFlow(test *framework.ClusterE2ETest, clusterOpts, clusterOpts2, clusterOpts3 []framework.ClusterE2ETestOpt) {
```

Comment:
Can we use variadic [function](https://gobyexample.com/variadic-functions) to make it more flexible if we wanted to do variable number of consecutive upgrades.?
```suggestion
func runInPlaceMultipleUpgradesFlow(test *framework.ClusterE2ETest, clusterOpts ...[]framework.ClusterE2ETestOpt) {
        test.CreateCluster()
	for _, opt := range clusterOpts {
	test.UpgradeClusterWithNewConfig(opt)
	test.ValidateClusterState()
	test.StopIfFailed()
}
	test.DeleteCluster()
}
```